src/wireviz/wv_html.py

```diff
@@ -35,14 +35,14 @@ def generate_html_output(
         # fall back to built-in simple template if no template was provided
         templatefile = Path(__file__).parent / "templates/simple.html"
 
-    html = open_file_read(templatefile).read()
+    html = file_read_text(templatefile)
 
     # embed SVG diagram (only if used)
     def svgdata() -> str:
         return re.sub(
             "^<[?]xml [^?>]*[?]>[^<]*<!DOCTYPE [^>]*>",
             "<!-- XML and DOCTYPE declarations from SVG file removed -->",
-            open_file_read(f"{filename}.tmp.svg").read(),
+            file_read_text(f"{filename}.tmp.svg"),
```

Comment:
This question might be slightly out of scope for this PR, but needs to be raised: Here, the SVG file is assumed to be utf-8 encoded, but it's not assumed at:
https://github.com/wireviz/WireViz/blob/close_files/src/wireviz/svgembed.py#L62-L64

What is correct? Should we assume utf-8 at both places, or does it really depend on what encoding is specified in the leading part of the SVG file itself?

Reply:
Good question. Since the HTML template contains `<meta charset="UTF-8">`, it would be best if the embedded SVG also was UTF-8. Not sure if we know or have control over what encoding Graphviz chooses to output.

Reply:
We probably should assume utf-8 and specify that encoding at both places. I also found a third place where that is already done:  https://github.com/wireviz/WireViz/blob/close_files/src/wireviz/Harness.py#L662

Then, we probably also should (as a minimum only where embedding SVG into HTML, or ideally at all places reading SVG) verify that the `encoding` property of the leading `xml` tag is either absent (utf-8 is default, I believe) or equal to any of the legal value variations that specify utf-8? If we detect a discrepancy, should we raise an exception or just print a warning that some characters might be rendered wrongly due to an unexpected `encoding` in the SVG file?

Reply:
TBH, I would prefer to keep it simple here and work with the assumption, in order to finish the release of v4.1.
Verification can be done as a separate feature/PR.

Reply:
I created #400 to follow up this issue. Please add your concerns and opinions there.

Reply:
OK thanks. I see some recent force-pushes so please re-request review once it's ready :)